crmsh/utils.py

```diff
@@ -1796,9 +1796,9 @@ def list_cluster_nodes(no_reg=False):
     node_list = []
     for node in cib.xpath(constants.XML_NODE_PATH):
         name = node.get('uname') or node.get('id')
+        # exclude remote node
         if node.get('type') == 'remote':
-            srv = cib.xpath("//primitive[@id='%s']/instance_attributes/nvpair[@name='server']" % (name))
-            if srv:
+            if cib.xpath(f"//primitive/instance_attributes/nvpair[@name='server' and @value='{name}']"):
```

Comment:
The attributes of `primitive` node should be check to ensure it is a `ocf::pacemaker:remote`.